{
  "error_message": "Route %s used \"%s\" inside a function cached with \"unstable_cache(...)\". Accessing Dynamic data sources inside a cache scope is not supported. If you need this data inside a cached function use \"%s\" outside of the cached function and pass the required dynamic data in as an argument. See more info here: https://nextjs.org/docs/app/api-reference/functions/unstable_cache",
  "file_path": "packages/next/src/server/route-modules/app-route/module.ts",
  "occurrence_count": 1
}
